Capri.Web/src/components/CardsView.vue

```diff
@@ -0,0 +1,422 @@
+                    thesisType: 'Master',
+                    studyType: 'Full time',
+                    freeSlots: 3,
+                },
+                {
+                    title: 'CaPri System',
+                    promoter: 'Jerzy Nawrocki',
+                    description:
+                        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce nulla lacus, facilisis vel felis id, porta blandit nulla. Quisque congue congue orci elementum tristique. Integer consequat est a nibh mollis aliquam. Integer leo justo, posuere fermentum euismod sit amet, blandit sagittis elit. Suspendisse laoreet massa nec neque tincidunt, in ullamcorper urna sollicitudin. Mauris vitae arcu bibendum, bibendum nisi vitae, aliquet nunc. In tincidunt purus sed lacus tincidunt facilisis. ',
+                    thesisType: 'Master',
+                    studyType: 'Full time',
+                    freeSlots: 3,
+                },
+                {
+                    title: 'CaPri System',
+                    promoter: 'Jerzy Nawrocki',
+                    description:
+                        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce nulla lacus, facilisis vel felis id, porta blandit nulla. Quisque congue congue orci elementum tristique. Integer consequat est a nibh mollis aliquam. Integer leo justo, posuere fermentum euismod sit amet, blandit sagittis elit. Suspendisse laoreet massa nec neque tincidunt, in ullamcorper urna sollicitudin. Mauris vitae arcu bibendum, bibendum nisi vitae, aliquet nunc. In tincidunt purus sed lacus tincidunt facilisis. ',
+                    thesisType: 'Master',
+                    studyType: 'Full time',
+                    freeSlots: 3,
+                },
+                {
+                    title: 'CaPri System',
+                    promoter: 'Jerzy Nawrocki',
+                    description:
+                        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce nulla lacus, facilisis vel felis id, porta blandit nulla. Quisque congue congue orci elementum tristique. Integer consequat est a nibh mollis aliquam. Integer leo justo, posuere fermentum euismod sit amet, blandit sagittis elit. Suspendisse laoreet massa nec neque tincidunt, in ullamcorper urna sollicitudin. Mauris vitae arcu bibendum, bibendum nisi vitae, aliquet nunc. In tincidunt purus sed lacus tincidunt facilisis. ',
+                    thesisType: 'Master',
+                    studyType: 'Full time',
+                    freeSlots: 3,
+                },
+            ],
+        };
+    }
+    public showDialog(value) {
```

Comment:
What's type of that `value`? Maybe it will be good to add interface for that type. I think you use that multiple times.